iris_markdown_EH.Rmd

```diff
@@ -0,0 +1,123 @@
+---
+title: "Ethan's Iris Analysis in RMarkdown"
+author: "Ethan Hill"
+date: "2020-01-16"
+output: html_document
+---
+
+Hello! Here are the plots and the code for the plots I have generated with the Iris dataset. I have also gone ahead and included the workflow I used to get the final plots...
+
+
+It is always a good habit to set your working directory:
+
+```{r}
+setwd("C:/Users/hille/Documents/Rbootcamp_S2020/Iris-repo/")
+```
+
+
+Next, I've scrambled the iris dataset (gotta make it harder) and stored the scrambled data into the object called mix:
+
+```{r}
+oo <- sample(x=1:150, 150, replace=FALSE)
+mix <- iris[oo,]
+```
+
+
+Once I have the scrambled data stored in the object called "mix" I can store each of the columns as a seperate vector:
+
+```{r}
+species <- mix$Species
+petalL <- mix$Petal.Length
+petalW <- mix$Petal.Width
+sepalL <- mix$Sepal.Length
+sepalW <- mix$Sepal.Width
+```
+
+
+Now that I have each of the columns stored as seperate vectors, I can take the species vector and assign each species its own unique color based on logical operations:
+
+```{r}
+species2 <- species
+color2 <- rep("red", 150)
+species2 == "setosa"
+color2[species2 == "setosa"] <- "purple"
+color2
+color2[species2 == "versicolor"] <- "blue"
+color2
+color2[species2 == "virginica"] <- "orange"
+color2
```

Comment:
Not wrong, but what is the purpose of species2? Does its value ever change? If not, you might want to cut this object.